openedx/features/course_experience/tests/views/test_course_outline.py

```diff
@@ -354,6 +349,27 @@ def visit_sequential(self, course, chapter, sequential):
         )
         self.assertEqual(200, self.client.get(last_accessed_url).status_code)
 
+    @override_switch(
+        '{}.{}'.format(
+            waffle.WAFFLE_NAMESPACE, waffle.ENABLE_COMPLETION_TRACKING
+        ),
+        active=True
+    )
+    def complete_sequential(self, course, sequential):
+        """
+        Completes provided sequential.
+        """
+        course_key = CourseKey.from_string(str(course.id))
+        # Fake a visit to sequence2/vertical2
+        block_key = UsageKey.from_string(unicode(sequential.location))
+        completion = 1.0
+        BlockCompletion.objects.submit_completion(
```

Comment:
I probably just don't actually know how completion works but I thought we only were submitting completion data for individual blocks right now?

Reply:
Discussed offline